pkg/front/errors.go

```diff
@@ -1,28 +1,34 @@
 // GENERATED BY textFileToGoConst
 // GitHub:     github.com/logrusorgru/textFileToGoConst
 // input file: html\front\errors.js
-// generated:  Wed Aug 31 16:07:24 CEST 2022
+// generated:  Thu Sep  8 01:55:07 CEST 2022
 
 package front
 
 const Errors = `const errorCodes = new Map([
-	['Wallet-0001', 'A wallet with the same nickname already exists on your computer'],
-	['Wallet-0002', 'Wrong wallet password'],
-	['Wallet-0003', 'Error while retrieving the specified wallet'],
-	['Wallet-1001', 'A wallet cannot be created without nickname'],
-	['Wallet-1002', 'A wallet cannot be created without password'],
-	['Wallet-1003', 'A technical issue occured while creating the wallet'],
-	['Wallet-2001', 'Specify the wallet wallet address to delete'],
-	['Wallet-2002', 'Error while trying to delete the wallet, may be opened by an other program'],
-	['Wallet-3001', 'Error while trying to import the wallet file'],
-	['Wallet-4001', 'Error while trying to fetch all your wallets'],
+	['Wallet-0001', 'That nickname is taken. Try Another'],
```

Comment:
@SlnPons can you check that ?

Reply:
You're right. Atm, there is not restriction about the name. But this will be implemented with TestNet 14 and so is supposed to be part of this iteration.

Reply:
Sorry, my request was unclear : can you check all the error messages :-)

Reply:
So as Solène already check that error messages, can we resolve that conversation ?